containers/dev/.devcontainer/entrypoint.sh

```diff
@@ -24,6 +24,20 @@ ansible --version &> /dev/null ||  if ! [ -z "${AVD_GITHUB_REPO}" ] && ! [ -z "$
   pip3 install -r ${HOME}/.ansible/collections/ansible_collections/arista/avd/requirements-dev.txt
 fi
 
+# In some cases AVD can not be correctly mounted, for ex. when running dev container as Codespace
+# In that case if collection is available in the container workspace, it will be installed from there
+CONTAINER_WORKSPACE=$(git rev-parse --show-toplevel)
+AVD_DEV_REQ_FILE_ALT="${CONTAINER_WORKSPACE}/ansible_collections/arista/avd/requirements-dev.txt"
+ansible --version &> /dev/null ||  if [ -f $AVD_DEV_REQ_FILE_ALT ]; then
+  sudo chown -R ${USERNAME} ${HOME}/.ansible
+  ANSIBLE_CORE_VERSION=$(cat ${AVD_DEV_REQ_FILE_ALT}| grep ansible-core)
+  pip3 install "${ANSIBLE_CORE_VERSION}"
```

Comment:
does it need to be `pip3` everywhere because the base container still has python2 or is just some sort of clarity?

Reply:
everything is Python3. All AVD dev containers are based on python-slim with 3.9, 3.10, 3.11 Python.